test/includes/common.sh

```diff
@@ -41,6 +41,16 @@ run_test() {
 httpd_create() {
     rm -rf httpd/mod_proxy_cluster /tmp/mod_proxy_cluster
     mkdir /tmp/mod_proxy_cluster
+    # make sure the native are cleaned before copying
+    for m in advertise mod_proxy_cluster balancers mod_manager
+    do
+        cd $m
```

Comment:
There is `native` missing, it should be `cd ../native/$m`.

Reply:
thanks fixed.